pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -67,6 +67,14 @@ const (
 	oldXFRMOutPolicyPriority = 50
 )
 
+type dir string
```

Comment:
Given the common meaning, `dir` I think would be better named `direction`.

Reply:
It took me a while to see what you were talking about :sweat_smile: You mean "directory", right? I guess that probably wouldn't be top of mind for someone working on the datapath. I'll change for extra clarity if other changes are requested.